### PR TITLE
adding a button style

### PR DESCRIPTION
**Pull request by clmntnbrn**
Adding a button style - @sashaward I know you've done some other work so I haven't added the grid and other styles, awaiting your pull request 😉

### CODE REVIEW

**Comment by clmntnbrn** [comment]
@sashaward now we need to learn how to resolve conflicts! Fun times!

**Comment by sashaward** [comment]
@clmntnbrn oh I love conflicts!

File: css/styles.css [hunk 1]
```diff
@@ -89,6 +100,29 @@ header {
...
 
+/******************************************
+/* BUTTONS
+/*******************************************/
+
+.btn {
+  -webkit-border-radius: 0;
+  -moz-border-radius: 0;
+  border-radius: 0px;
+  font-family: Arial;
```
Comment (barwick) [inline css/styles.css]: Tsk tsk

**Review by barwick — CHANGES_REQUESTED** [review]
Please can you use the same font

File: css/styles.css [hunk 2]
```diff
@@ -76,9 +78,18 @@ html {
 /******************************************
 /* TYPOGRAPHY
 /*******************************************/
-
+<<<<<<< HEAD
```
Comment (sashaward) [inline css/styles.css]: 😱

**Review by sashaward — APPROVED** [review]
✅

**Pull request merged**